src/displayBackend/wayland/Surface.cpp

```diff
@@ -115,6 +116,12 @@ void Surface::clear()
 	wl_surface_commit(mWlSurface);
 }
 
+void Surface::handleSurfaceConfiguration(xdg_surface *xdg_surface, uint32_t serial)
+{
+	unique_lock<mutex> lock(mMutex);
```

Comment:
Add empty line after lock. The rule is separate var definition from operators.

Reply:
done